gcloud/credentials.py

```diff
@@ -207,6 +217,9 @@ def _get_signature_bytes(credentials, string_to_sign):
         pkey = _get_pem_key(credentials)
         if not isinstance(string_to_sign, six.binary_type):
             string_to_sign = string_to_sign.encode('utf-8')
+        if crypto is None:
+            raise EnvironmentError('pyOpenSSL must be installed to sign '
```

Comment:
Weird how? The content doesn't fit on a single line. To be clear, you want me to go from

```python
            raise EnvironmentError('pyOpenSSL must be installed to sign '
                                   'content using a private key')
```

to

```python
            raise EnvironmentError(
                'pyOpenSSL must be installed to sign content using a '
                'private key')
```